concent_api/utils/logging.py

```diff
@@ -11,35 +11,40 @@
 def log_message_received(message: Message, client_public_key: str):
     task_id = get_task_id_for_logging(message)
     subtask_id = get_subtask_id_for_logging(message)
-    logger.info('A message has been received in `send/` -- MESSAGE_TYPE: {} -- TASK_ID: {} -- SUBTASK_ID: {} -- CLIENT PUBLIC KEY: {}'.format(
+    logger.info('A message has been received in `send/` -- MESSAGE: {} --TYPE: {} -- TASK_ID: {} -- SUBTASK_ID: {} -- CLIENT PUBLIC KEY: {}'.format(
+        type(message).__name__,
         message.TYPE,
         task_id,
         subtask_id,
         client_public_key,
     ))
-    logger.debug('A message has been received in `send/` -- MESSAGE: {}'.format(
-        message,
+    logger.debug('A message has been received in `send/` -- MESSAGE: {} --TYPE: {}'.format(
```

Comment:
Here we wanted to print whole message, please revoke this change.

Reply:
Done